__main__.py

```
@@ -0,0 +1,610 @@
+                    {"plugin": "RCEExploitPlugin", "why": "Remote code execution",
+                     "params": {"establish_shell": True}},
+                ])
+            llm = LocalFunctionConnector(generate_fn=aggressive_llm)
+            print("[+] Local function LLM configured")
+            
+        else:
+            print(f"[!] Unknown LLM type: {args.llm}")
+            return None
+        
+        bot.set_llm(llm)
+        return llm
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)